source/sass/blocks/checkbox.scss

```
@@ -17,3 +21,43 @@
     color: v.$color-gray-400;
   }
 }
+
+.checkbox__input {
+  display: none;
+}
+
+.checkbox__input + .checkbox__text::before {
```

Вроде можно обойтись тут и более простым селектором же )
```suggestion
.checkbox__text::before {
```

Через `+` нам надо выбирать обязательно только из-за того, что нужно проверить состояние